### PR TITLE
docs(core): improve intro page

### PR DESCRIPTION
**Pull request by juristr**
Working on improvements for the intro page: https://nx-dev-git-fork-juristr-docs-intro-section-update-nrwl.vercel.app/getting-started/intro

### CODE REVIEW

File: docs/shared/getting-started/intro.md [hunk 1]
```diff
@@ -1,8 +1,18 @@
...
+
+- **Efficient Task Execution**: Nx runs tasks in parallel, keeping potential dependencies among them in mind
+- **Local & Remote Caching**: With local and remote caching, Nx prevents unnecessary re-runs of tasks, saving you valuable dev time.
+- **Smooth Code Migrations**: Leveraging Nx plugins means gaining automatic code generation and migration tools - useful for keeping NPM packages up-to-date and minimizing breaking changes.
+- **Editor Integration**: Nx ships with Nx Console, an extension for VSCode & IntelliJ (and an LSP for you Vim nerds out there)
+- **Make it Your Own**: The [Nx Devkit](/plugins/intro/getting-started) lets you extend Nx's capabilities with your own automation, or you can even build your CLI on top of it.
+
+Find out more about [why you should use Nx](/getting-started/why-nx) or browse our [core features](/core-features) section.
+
+## Try Yourself!
```
Comment (meeroslav) [inline docs/shared/getting-started/intro.md:15]:
```suggestion
## Try it yourself!
```

**Review by meeroslav — APPROVED** [review]
Left one comment, but I generally like it.

File: docs/shared/getting-started/intro.md (hunk 1 above)
Comment (juristr) [reply docs/shared/getting-started/intro.md:15]: I believe you can also say "Try Yourself"... Not sure whether if you add "it" you then need to say "Try it by yourself"

Comment (juristr) [reply docs/shared/getting-started/intro.md:15]: Yep, this is what I intended. That's why I went for "Try Yourself" :)

image

**Comment by juristr** [comment]
pushing up multiple smaller PRs based off this one

**Pull request closed without merging**